source/slang/slang-lower-to-ir.cpp

```diff
@@ -6087,6 +6408,7 @@ struct DeclLoweringVisitor : DeclVisitor<DeclLoweringVisitor, LoweredValInfo>
         return as<IRStringLit>(builder->getStringValue(stringLitExpr->value.getUnownedSlice()));
     }
 
+#if 0
```

Comment:
Should we just remove them?

Reply:
Yes. I failed to clean these up after the merge. I will plan to do a bit more cleanup before calling this done.